plugins/engines/ospray/OSPRayEngine.cpp

```diff
@@ -379,7 +379,7 @@ void OSPRayEngine::_createCameras()
     PropertyMap::Property eyeSeparation{"interpupillaryDistance",
                                         "Eye separation",
                                         0.0635f,
-                                        {0.f, 10.f}};
+                                        {0.f, 1e31f}};
```

Comment:
Before doing these kind of changes to params that are dynamically rendered by the UI, please test the UI as well! As @tribal-tec mentioned, this will make the sliders and input controls for this property unusable.

I advise reverting this commit and finding a different solution to whatever problem this was trying to solve.

Reply:
Another option is to remove the limits here, so no slider component should be used then if I'm not mistaken.

Reply:
Nope, any property that is `{'type': "number"'}` will have a slider. But regardless of that, the input will still jump a high amount when using the keyboard arrows or mouse wheel.

Reply:
This requires careful thinking I'm afraid. You may have a similar problem with simulation timestamps and simulation playback speed for very long simulations. I don't remember seeing any integrated UI component that allows choosing a number with high precision when the limits are very large... The only solution that I can think of is a bit hacky and complexifies the UI. The idea would be to detect that the range is very large (unless a hint is provided from json) and split the slider in two, one for the significand and other for the order of magnitude. Another solution is to have a normal slider that goes from 0 to 1000 and a combo box for the multiplier (u, m, 1, K, M, ...)